lib/libzfs/libzfs_util.c

```diff
@@ -1940,3 +1941,91 @@ zprop_iter(zprop_func func, void *cb, boolean_t show_all, boolean_t ordered,
 {
 	return (zprop_iter_common(func, cb, show_all, ordered, type));
 }
+
+static void
+nomem(void)
+{
+	(void) fprintf(stderr, gettext("internal error: out of memory\n"));
+	exit(1);
+}
+
+/*
+ * Utility function to guarantee malloc() success.
+ */
+
+static void *
+safe_malloc(size_t size)
+{
+	void *data;
+
+	if ((data = calloc(1, size)) == NULL)
+		nomem();
+
+	return (data);
+}
+
+#ifndef ZFS_VERSION_EXTRA
+#define ZFS_VERSION_EXTRA ""
+#endif
+/*
+ * Returns the zfs userland version with ZFS_META_RELEASE and ZFS_META_GITREV
```

Comment:
This comment doesn't seem to match the code, which returns ZFS_META_GITREV and ZFS_VERSION_EXTRA.  But I think we should be printing something like ZFS_META_RELEASE (perhaps in addition to ZFS_META_GITREV).

Reply:
From the looks of it, ZFS_META_GITREV pretty much was an all inclusive version string and I had removed the others and just used it exclusively.  Did I miss something?

Reply:
I think you're right, ZFS_META_GITREV incorporates the others, so I think you're good, just update the comment.